### PR TITLE
CNV: provide a multus temporary workaround

### PR DESCRIPTION
**Pull request by karmab**
addresses https://github.com/openshift-kni/install-scripts/issues/123

### CODE REVIEW

**Comment by e-minguez** [comment]
LGTM but I would like some comment/ideas as even if it is a workaround that eventually will be removed it feels _dirty_ (I know I know...)

/cc @sreichar @russellb

File: CNV/fix-multus.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+cat << EOF > 80-openshift-network.conf
+{ "name": "openshift-sdn", "type": "multus", "namespaceIsolation": true, "logLevel": "verbose", "kubeconfig": "/etc/kubernetes/cni/net.d/multus.d/multus.kubeconfig", "delegates": [ { "cniVersion": "0.3.1", "name": "openshift-sdn", "type": "openshift-sdn" } ] }
+EOF
+
+for node in $(oc get nodes -o jsonpath="{.items[*].metadata.name}"); do
+  scp 80-openshift-network.conf core@$node:/tmp/
+  ssh core@$node "sudo mv /tmp/80-openshift-network.conf /etc/kubernetes/cni/net.d/80-openshift-network.conf"
```
Comment (sreichar) [inline CNV/fix-multus.sh]: Any concerns about ownership, mode, or selinux context.

**Comment by sreichar** [comment]
[test]

**Comment by karmab** [comment]
this is dirty indeed but while we wait, it s necessary for people to be able to run vms related workflows

**Comment by sreichar** [comment]
[test]

**Comment by johnbieren** [comment]
[test]

**Comment by johnbieren** [comment]
[test]

**Pull request merged**